rfcs/code_of_conduct.md

```diff
@@ -0,0 +1,54 @@
+# RFC #54: Code of Conduct
+
+## Summary
+
+Introduce a code of conduct which clearly defines the acceptable behavior for
+all participants in the WPT project and includes contact information for a
+group of individuals who can be trusted to enforce the policy.
+
+Publishing a code of conduct is a way "to be overt in our openness, welcoming
+all people to contribute, and pledging in return to value them as whole human
+beings and to foster an atmosphere of kindness, cooperation, and
+understanding." (via
+[contributor-covenant.org](https://www.contributor-covenant.org/)
+
+## Details
+
+This RFC consists of two parts: ratifying the content in the [proposed
+`CODE_OF_CONDUCT.md`](https://github.com/web-platform-tests/wpt/pull/7520)
```

Comment:
Our existing text is based on an edited version of the Rust CoC, itself based on an old version of the Contributor Covenant. Have we considered whether we just want to base it upon the latest Contributor Covenant text? (This is purely a comment/open question, I don't know how much has changed.)

Reply:
Assuming https://www.contributor-covenant.org/version/2/0/code_of_conduct/is the new Contributor Covenant, I took a look through. I didn't find many substantive differences (there are a few). The Contributor Covenant arguably has a more structured layout, but I found their 'Enforcement Guidelines' section actually a little confusing; I wasn't clear if they were specific examples or just general 'impact' statements.

I lean towards what we have being 'good enough', but I'm definitely open to being wrong about that. I don't have much experience in this space.